blog/2024-10-8-self-hosting-reflex-with-docker.md

```diff
@@ -0,0 +1,339 @@
+---
+author: Tom Gotsman
+date: 2024-10-08
+title: Self Hosting a Reflex app with Docker
+description: Learn how to deploy and self-host your Reflex application using Docker for efficient containerization.
```

Comment:
Hosting Reflex on your own infra using Docker for efficient containerization.

Reply:
changed